TheEthicalHackerCup/Assets/MainScreen/Scripts/PassiveAttacks/PassiveAttackControl.cs

```diff
@@ -25,7 +25,7 @@ void Start()
     void Update()
     {
         // Every 30 seconds a random passive attack will become available for a random amount of time
-        if (Time.time - timeSinceLastAttack > 30) {
+        if (Time.time - timeSinceLastAttack > 25) {
```

Comment:
Would making the time between passives dynamic be valuable? I would be reasonably easy to replace that hardcoded 25 with an integer property with get/set.

I think progressing from longer to shorter in-between-passive-attack-times would be something helpful to the player as there this would give us another way to balance the opp knowledge economy.

Reply:
WIll leave it as hardcoded for now but will add TODO for updating this to be  held in the singleton